R/build-home-index.R

```diff
@@ -116,8 +116,23 @@ data_home_sidebar <- function(pkg = ".") {
 
 }
 
-default_sidebar_structure <- function() {
-  c("links", "license", "community", "citation", "authors", "dev")
+default_sidebar_structure <- function(all = FALSE) {
+  components <- tibble::tribble(
+    ~name, ~definition, ~default,
+    "links", "Useful links", TRUE,
```

Comment:
I think it's better to keep the description in the docs, and write it in such a way that you can describe the default values/order in a separate a sentence.